### ISSUE TITLE
stripPrefix option removes or adds http:// always.

### ISSUE THREAD

**Issue by therealmarv** [gregjacobs/Autolinker.js#73]
Is it possible to have a third option or another option instead of `stripPrefix`? At the moment [stripPrefix](http://gregjacobs.github.io/Autolinker.js/docs/#!/api/Autolinker-cfg-stripPrefix) set at true or false either removes http:// or adds http:// for all links and this changes the original text slightly!

Here is an example...

Input text:
```
http://hereissomelink.com
thisisalsoalink.com
```

Output with `stripPrefix: false`:
```html
<a href="http://hereissomelink.com" target="_blank">http://hereissomelink.com</a>
<a href="http://thisisalsoalink.com" target="_blank">http://thisisalsoalink.com</a>
```

Output with `stripPrefix: true`:
```html
<a href="http://hereissomelink.com" target="_blank">hereissomelink.com</a>
<a href="http://thisisalsoalink.com" target="_blank">thisisalsoalink.com</a>
```

But I want and need is that the original text will not be changed so I only get the original text but linkified:
```html
<a href="http://hereissomelink.com" target="_blank">http://hereissomelink.com</a>
<a href="http://thisisalsoalink.com" target="_blank">thisisalsoalink.com</a>
````

also adding `www` or removing `www` is something which changes text too much for my usage.

Thanks

**Comment by mulhoon** [comment]
+1 I think this is really important to keep the original text, especially if you want to create links while the user is typing. I think it should `keepOriginalText` by default.

I had a dig around to fix it but I broke some of the tests :(

**Comment by petko** [comment]
+1 I am new Autolinker.js and I was searching for exactly the same option!

**Comment by LeZuse** [comment]
:+1: Also looking for something like this. `keepOriginalText` seems like would do the job. But I am not so sure about making it true by default, since that might break backward compatibility.

**Comment by albv** [comment]
+1

**Comment by gregjacobs** [comment]
Hey all. Long awaited, I know.. Decided to just keep the original matched text by default. Fixed in v0.21.0
